src/ec/packages/ec-component-footer-standardised/footer-standardised.html.twig

```diff
@@ -0,0 +1,99 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "sections" (array of objects) (default: []):
+    - "title" (optional) (string) OR (object with Link component in property)
+    - "title_class_name" (optional) (string) (default: '')
+    - "description" (optional) (string) (default: '')
+    - "content_before" (optional) (string) (default: '')
+    - "list_class_name" (optional) (string) (default: '')
+    - "links" (optional) (array of Link components) (default: []):
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'     
+#}
+
+{# Internal properties #}
+{% set _sections = sections|default([]) %}
+
+{% set _css_class = '' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<footer class="ecl-footer-standardised"{{ _extra_attributes|raw }}>
+  {# Main container #}
+  <div class="ecl-container ecl-footer-standardised__container">
+    {% for section in _sections %}
+      {# Section #}
+      <div class="ecl-footer-standardised__section {{ 'ecl-footer-standardised__section' ~ loop.index }}">
+        {# content before #}
+        {% if section.content_before is defined and section.content_before is not empty %}
+         <div class="ecl-footer-standardised__content">{{ section.content_before|e }}</div>
```

Comment:
I used the escaped filter in case of quotes into the text. But indeed it works with or without the escape filter. Could be an escaping filter somewhere else... By the way it costs nothing to let it.